CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone
+# opens a pull request.
+*       @blootsvoets @nivemaham @fnobilia
```

Comment:
AFAIU all tree accounts here will be involved in the review.

Quoting [Understanding the GitHub Flow](https://guides.github.com/introduction/flow/)

>Pull Requests initiate discussion about your commits. Because they're tightly integrated with the underlying Git repository, anyone can see exactly what changes would be merged if they accept your request. You can open a Pull Request at any point during the development process: when you have little or no code but want to share some screenshots or general ideas, when you're stuck and need help or advice, or when you're ready for someone to review your work. By using GitHub's @mention system in your Pull Request message, you can ask for feedback from specific people or teams, whether they're down the hall or ten time zones away.

I think the review process should start before merging to `dev`. At that time the developer is still in time to discuss and modify the current fix, instead of waiting till merging into `master`, merging that may happen a little bit later on. Furthermore, it is easier commenting on single change instead of committing of several changes at the same time.